apps/meteor/client/views/admin/apps/BundleChips.tsx

```diff
@@ -22,7 +22,7 @@ const BundleChips = ({ bundledIn, isIconOnly }: BundleChipsProps): ReactElement
 	return (
 		<>
 			{bundledIn.map((bundle) => (
-				<>
+				<Fragment key={bundle.bundleId}>
```

Comment:
Comment for line 41:
try using our typography tokens from [fuselage](http://localhost:6006/?path=/docs/layout-box-props--font-sizes) to keep the standards.
In this case would be **c2**

Reply:
Solved